tests/lib/rules/jsx-wrap-multilines.js

```diff
@@ -155,14 +217,14 @@ ruleTester.run('jsx-wrap-multilines', rule, {
       code: DECLARATION_TERNARY_PAREN
     }, {
       code: DECLARATION_TERNARY_NO_PAREN,
-      options: [{declaration: false}]
+      options: [{condition: false}]
     }, {
       code: ASSIGNMENT_TERNARY_SINGLE_LINE
     }, {
       code: ASSIGNMENT_TERNARY_PAREN
     }, {
       code: ASSIGNMENT_TERNARY_NO_PAREN,
-      options: [{assignment: false}]
+      options: [{condition: false}]
```

Comment:
why does this option need to change?

Reply:
This test case checked the condition in assignment. Now checking the condition is separated.

Reply:
Similarly, does that mean that "assignment: true" now doesn't check as much as it used to? That would make this a breaking change.

Reply:
Yes, it is a broken change for assignments too